src/main/resources/application.conf

```diff
@@ -1,8 +1,28 @@
 
 
-h2mem1 = {
-  url = "jdbc:h2:mem:test1"
-  driver = org.h2.Driver
-  connectionPool = disabled
-  keepAliveConnection = true
-}
\ No newline at end of file
+database {
+  config = h2mem1
+
+  h2mem1 = {
+    url = "jdbc:h2:mem:test1"
```

Comment:
Not sure. This was a PR into a PR refactoring to add MySQL support at the request of Jeff. Feel free to ask over on the original PR #7.